app.rb

```diff
@@ -40,28 +71,79 @@ def load_books
 
   def load_data
     load_labels
+    load_genres
     load_books
+    load_albums
   end
 
+  # Option 5:
   def create_label(title, color)
     @labels << Label.new(title, color)
   end
 
+  # Option 6:
+  def create_genre
+    puts "Name of the genre: \n"
+    name = gets.chomp
+    @genres << Genre.new(name)
+    puts "Genre #{name} succesfully created!"
+  end
+
+  # Option 8:
+  def create_album
+    puts "Genre: \n"
```

Comment:
I think when you use puts to print, it automatically adds a new line. Maybe de `\n` can be omitted.